composer.json

```diff
@@ -1,4 +1,5 @@
 {
+    "name": "sandreas/m4b-tool",
     "minimum-stability": "dev",
     "prefer-stable": true,
     "platform":{"php":"7.4"},
```

Comment:
nixpkgs runs it with 8.2; this caused a couple issues which I solved. There may be more though.